conanfile.py

```diff
@@ -31,7 +31,7 @@ class Clio(ConanFile):
         'protobuf/3.21.9',
         'grpc/1.50.1',
         'openssl/1.1.1v',
-        'xrpl/2.5.0-b1',
+        'xrpl/2.5.0-b1@my/feature',
```

Comment:
Will change this when `rippled` PR gets merged and new tag is created

Reply:
there's a confluence page "How to deploy `libxrpl` to Artifactory", and you can add your custom libxrpl.  This allow CI to pass using custom libxrpl of a rippled branch that hasn't made a beta release yet.